test/E2ETest/ApplicationFactoryTests.cs

```diff
@@ -27,11 +29,12 @@ public async Task CyclesStopImmediately()
 - name: test-project
   include: tye.yaml";
             var yamlFile = Path.Combine(projectDirectory.DirectoryPath, "tye.yaml");
-
             await File.WriteAllTextAsync(yamlFile, content);
+
             // Debug targets can be null if not specified, so make sure calling host.Start does not throw.
             var outputContext = new OutputContext(_sink, Verbosity.Debug);
-            var application = await ApplicationFactory.CreateAsync(outputContext, new FileInfo(yamlFile));
+            var projectFile = new FileInfo(yamlFile);
```

Comment:
What's the point of this change?

Reply:
Do you want me to rollback that change ?

Reply:
Refactoring is fine, and I do see where this change is necessary for tests to work. But in this particular test and several like it, there's no reason to split the instantiation of the project file into a separate line. Usually we try to avoid unnecessary churn in the code, but given it doesn't affect and functionality, I'm okay with keeping this change. Though I would say in general, if there's no functional benefits, try to avoid churn.

Reply:
noted, so far I'll keep it like this just tell me, anytime in this PR lifetime if you want it inlined

Reply:
No need, I'm mainly adding the note for when you make contributions in the future! 😄